app/views/idv/phone/new.html.erb

```diff
@@ -61,6 +61,7 @@
     ) do |f| %>
   <%= render PhoneInputComponent.new(
         form: f,
+        confirmed_phone: @idv_form.phone.present?,
```

Comment:
Another edge case worth noting is that the value won't be assigned if the user has _multiple_ confirmed phones:

https://github.com/18F/identity-idp/blob/81f0e6787079d507313f6c1b10e367e1b69c9116/app/forms/idv/phone_form.rb#L26

But also, practically speaking, we already restrict this instance to US-only, so I guess we don't really have much to worry about on this front anyways?

Reply:
Ah, that's a very good catch. Maybe we drop the parameter for this and the "send link" forms so it falls back to the more permissive default?

Reply:
> Ah, that's a very good catch. Maybe we drop the parameter for this and the "send link" forms so it falls back to the more permissive default?

Yeah, I'd be okay with that. We'll need to add that permissive default though, since right now it's still a required keyword.